src/control/cmd/dmg/pretty/printers_test.go

```diff
@@ -107,3 +107,44 @@ host1:1,host2:2 whoops
 		})
 	}
 }
+
+func TestControl_UpdateErrorSummary(t *testing.T) {
+	for name, tc := range map[string]struct {
+		resp      *control.CollectLogResp
+		cmd       string
+		expStdout string
+	}{
+		"empty response": {
+			resp:      new(control.CollectLogResp),
+			cmd:       "empty",
+			expStdout: ``,
+		},
+		"one host error": {
+			cmd: "hostname",
+			resp: &control.CollectLogResp{
+				HostErrorsResp: control.MockHostErrorsResp(t,
+					&control.MockHostError{
+						Hosts: "host1",
+						Error: "command not found",
+					}),
+			},
+			expStdout: `
+Hosts Command  Error             
+----- -------  -----             
+host1 hostname command not found 
+`,
+		},
```

Comment:
please add a concrete response unit test case (`&control.CollectLogResp{}`), nil case, one for the same error occurring on multiple hosts as well as multiple different errors on the same host.

Reply:
Added few more test cases.